tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py27,py33
```

Comment:
So it looks possible.  However, supporting multiple minor versions of Python with the same major version (i.e. 3.4 and 3.5) might prove to be a little more complicated and have some OS specific steps, assuming one hasn't compiled from source to a directory such as `/opt/`